package.json

```diff
@@ -14,6 +14,8 @@
   "author": "OpenZipkin <openzipkin.alt@gmail.com>",
   "repository": "https://github.com/openzipkin/zipkin-js",
   "devDependencies": {
+    "babel-cli": "^6.18.0",
+    "babel-preset-es2015": "^6.18.0",
```

Comment:
I think we should be able to get away with only using babel-preset-env and then target node 4.